liquid-integration/src/main/java/com/e/liquid_integration/dialog/CountrySupportDialog.kt

```diff
@@ -0,0 +1,42 @@
+package com.e.liquid_integration.dialog
+
+import android.app.Dialog
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
+import com.e.liquid_integration.R
+import com.e.liquid_integration.data.LiquidConstants
+import com.e.liquid_integration.ui.LiquidLoginActivity
+
+class CountrySupportDialog(val _context: Context) : Dialog(_context, R.style.Theme_Dialog) {
```

Comment:
@tomasz-ludek What is your opinion on underscores?

Reply:
@HashEngineering @tomasz-ludek I added some comments asking them to remove it to match the rest of the project.

Reply:
yes i have removed underscores. Thanks